velox/docs/functions/spark/math.rst

```diff
@@ -233,3 +233,10 @@ Mathematical Functions
 .. spark:function:: unaryminus(x) -> [same as x]
 
     Returns the negative of `x`.  Corresponds to Spark's operator ``-``.
+
+.. spark:function:: unhex(x) -> varbinary
+
+    Converts hexadecimal ``x`` to varbinary.
+    If ``x`` has odd number of characters, first character will be parsed first and putted in index 0,
+    remaining characters continue be parsed and putted start from index 0.
```

Comment:
I remember Masha once suggested me to use active verb in the doc, like "if xxx, parses xxx, then continues xxx."
My suggestion:
```
If input size is odd, parses the first character and puts it at index 0 of binary result.
Then, continues parsing remaining characters, if existing, and
still puts the parsing result starting from index 0, which will overwrite the previous value.
```